src/app/containers/Video/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Helmet from 'react-helmet';
 import Figure from '@bbc/psammead-figure';
+import Iframe from 'react-iframe';
```

Comment:
If you're using iframes on AMP, you need to use the `amp-iframe` component instead of a standard iframe. https://amp.dev/documentation/components/amp-iframe

You might also want to look into https://amp.dev/documentation/components/amp-video-iframe as well for greater access to video features.